cmd/endpoint-ellipes.go

```diff
@@ -0,0 +1,301 @@
+/*
```

Comment:
notice typo in file name -suggest changing to cmd/endpoint-ellipse.go

Reply:
Fixed